### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by jwulf**
* Add docker-compose instructions
* Add style code note to CONTRIBUTING.md

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 96cf7757a540f942ed000b42ff5988b2544aabd9
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

File: docs/src/getting-started/tutorial-setup.md [hunk 1]
```diff
@@ -15,7 +15,9 @@ If you're looking for a very fast (but less comprehensive) "first contact" exper
 
 The tutorial assumes you have some basic knowledge of what Zeebe is and what it's used for. If you're completely new to Zeebe, you might find it helpful to read through the ["What is Zeebe?"](https://docs.zeebe.io/introduction/what-is-zeebe.html) docs article first.
 
-Below are the components you'll use in the tutorial. Please download the full distributions for these components instead of running them with Docker.
+Below are the components you'll use in the tutorial. The easiest way to run them is to use the `operate` docker-compose profile in the [zeebe-docker-compose](https://github.com/zeebe-io/zeebe-docker-compose) repository.
```
Comment (wints) [inline docs/src/getting-started/tutorial-setup.md]: Even if using docker-compose, is it still necessary to download the Modeler distribution?

**Review by wints — CHANGES_REQUESTED** [review]
@jwulf I think it would also make sense to update the article pages with more info for Docker users when we discuss how to deploy a workflow model, create instances and workers, send messages, etc. 

I don't have much experience with Docker and therefore am not sure where to go to complete these steps, whereas it's more straightforward when just running the distributions–some quick guidance here for other new-to-Docker users would he helpful, especially since we call that the recommended approach.

File: docs/src/getting-started/tutorial-setup.md (hunk 1 above)
Comment (jwulf) [reply docs/src/getting-started/tutorial-setup.md]: I've updated the tutorial.

Comment (jwulf) [reply docs/src/getting-started/tutorial-setup.md]: WDYT?

**Review by wints — APPROVED** [review]
LGTM, thanks @jwulf!

**Comment by menski** [comment]
I cherry-picked these changes to master and pushed them live

**Comment by wints** [comment]
Thanks much, @menski 👏

**Pull request merged**